lm/srilm.py

```diff
@@ -111,6 +188,7 @@ def __init__(
         count_exe: tk.Path,
         *,
         vocab: Optional[tk.Path] = None,
+        discounts: Optional[tk.Path] = None,
```

Comment:
This is a hash breaking change, you would need a `__sis_hash_exclude__` for that.